extensions/content-usage-metrics/README.md

```diff
@@ -0,0 +1,46 @@
+# Content Usage Metrics
+
+A Shiny app that helps you understand content usage patterns on your Connect server. Browse an overview of usage across your published content, and dive into a detailed breakdown of visit-level data for each piece of content.
```

Comment:
May want to match with the description in the `manifest.json` based on any changes you make in this PR.

Reply:
Yep